Code/Framework/AzCore/AzCore/IO/Streamer/FileRequest.cpp

```diff
@@ -12,117 +12,124 @@
 #include <AzCore/IO/Streamer/FileRequest.h>
 #include <AzCore/IO/Streamer/StreamerContext.h>
 
-namespace AZ::IO
-{
-    //
-    // Command structures.
-    //
-
-    FileRequest::ExternalRequestData::ExternalRequestData(FileRequestPtr&& request)
-        : m_request(AZStd::move(request))
-    {}
+//
+// Command structures.
+//
 
-    FileRequest::RequestPathStoreData::RequestPathStoreData(RequestPath path)
-        : m_path(AZStd::move(path))
-    {}
-
-    FileRequest::ReadRequestData::ReadRequestData(RequestPath path, void* output, u64 outputSize, u64 offset, u64 size,
-        AZStd::chrono::system_clock::time_point deadline, IStreamerTypes::Priority priority)
-        : m_path(AZStd::move(path))
-        , m_allocator(nullptr)
-        , m_deadline(deadline)
-        , m_output(output)
-        , m_outputSize(outputSize)
-        , m_offset(offset)
-        , m_size(size)
-        , m_priority(priority)
-        , m_memoryType(IStreamerTypes::MemoryType::ReadWrite) // Only generic memory can be assigned externally.
-    {}
-
-    FileRequest::ReadRequestData::ReadRequestData(RequestPath path, IStreamerTypes::RequestMemoryAllocator* allocator,
-        u64 offset, u64 size, AZStd::chrono::system_clock::time_point deadline, IStreamerTypes::Priority priority)
-        : m_path(AZStd::move(path))
-        , m_allocator(allocator)
-        , m_deadline(deadline)
-        , m_output(nullptr)
-        , m_outputSize(0)
-        , m_offset(offset)
-        , m_size(size)
-        , m_priority(priority)
-        , m_memoryType(IStreamerTypes::MemoryType::ReadWrite) // Only generic memory can be assigned externally.
-    {}
-
-    FileRequest::ReadRequestData::~ReadRequestData()
-    {
-        if (m_allocator != nullptr)
+namespace AZ::IO::Requests
+{
+ReadData::ReadData(void* output, u64 outputSize, const RequestPath& path, u64 offset, u64 size, bool sharedRead)
```

Comment:
This namespace is missing an indent on it's contents

Reply:
+1 (using a selection with `Ctrl-K, F` in Visual Studio will run clang-format to fix this too 🙂)